src/hotspot/share/gc/shared/satbMarkQueue.hpp

```diff
@@ -115,7 +115,7 @@ class SATBMarkQueueSet: public PtrQueueSet {
 
   // Return true if the queue's buffer should be enqueued, even if not full.
   // The default method uses the buffer enqueue threshold.
-  virtual bool should_enqueue_buffer(SATBMarkQueue& queue);
+  bool should_enqueue_buffer(SATBMarkQueue& queue);
```

Comment:
Why drop `virtual` here? Is it because Shenandoah was the only virtual override of it, and now we can do the non-virtual call?

Reply:
Yes. IIRC we introduced that when we upstreamed Shenandoah, and can drop it again, thus restoring the original non-virtual version.

Reply:
Okay then!